shared-libs/validation/test/validations.js

```diff
@@ -197,7 +193,7 @@ describe('validations', () => {
     return validation.validate(doc, validations).then(errors => {
       assert.deepEqual(errors, [
         {
-          code: 'invalid_xyz_unique',
+          code: 'invalid_xyz',
```

Comment:
This is the other potential regression - medic validation errors used to append the validation rule name on to the code. This was to make them work alongside pupil validations. Now that they're merged this is no longer required, and standardising on the code makes sense. However, if anyone relied on the validation code this would be a regression, but I doubt anyone is as it _should_ be an implementation detail and nothing more.